pkg/splunk/enterprise/configuration.go

```diff
@@ -247,6 +251,12 @@ func getSplunkPorts(instanceType InstanceType) map[string]int {
 	}
 
 	switch instanceType {
+	case SplunkMonitoringConsole:
+		result["dfccontrol"] = 17000
```

Comment:
Do we need DFS ports on MC?

Reply:
we can actually avoid this here since, in theory, it is like a search head but we will not be running DFS searches/service on it